packages/babel-preset-env/data/overlapping-plugins.js

```diff
@@ -4,7 +4,8 @@ module.exports = new Map();
 
```

Comment:
Could we remove this file since it is _not_ used?

Reply:
I would prefer to keep it, since it's exactly what we will need when migrating the plugins from `@babel/preset-modules` into this preset.